app/manifest/_base.json

```diff
@@ -1,5 +1,5 @@
 {
-  "author": "https://metamask.io",
+  "author": "https://leucine0.blockscout.alphacarbon.network/",
```

Comment:
- Should not be the link to the testnet
- Will confirm with Eric what should we put here